lib/pipeline/orcabus-stateful-pipeline-stack.ts

```diff
@@ -148,6 +148,9 @@ class OrcaBusStatefulDeploymentStage extends cdk.Stage {
   ) {
     super(scope, environmentName, { env: { account: env?.account, region: 'ap-southeast-2' } });
 
-    new OrcaBusStatefulStack(this, 'OrcaBusStatefulStack', stackProps.orcaBusStatefulConfig);
+    new OrcaBusStatefulStack(this, 'OrcaBusStatefulStack', {
+      env: env,
+      ...stackProps.orcaBusStatefulConfig,
+    });
   }
```

Comment:
Perhaps we could just put new pipe stack here instead inside of the  stateful stack?

Reply:
From my side, could I leave refactoring `OrcaBusStatefulStack` with you, Will. You are the CodePipeline expert - so, I wouldn't want to mess up the setup done there.

But, yah, Flo is having a little teething issue with missing alembic, the `env`. We both puzzle the other days.

Reply:
I could move my stack here, yes. 
However, as Victor said, I don't want to interfere with your plans. I thought the `OrcaBusStatefulStack` is supposed to become a wrapper class instead of a stack. 

Either way, happy to go with what works best. Here I just point out one of the "issues" I ran into.